hicommonwealth/commonwealth
# Move refresh logic to script

## Link to Issue
Closes: #5641

## Description of Changes
- Moves membership refresh logic to a script
- Removes route

## Test Plan
- Run `yarn refresh-all-memberships
- Confirm that refresh runs correctly, shows in logs

## Deployment Plan
Create new dyno for script

## Other Considerations
N/A
